.env.example

```diff
@@ -1,2 +1,3 @@
 NEXT_PUBLIC_SUPABASE_ANON_KEY=SUPABASE_ANON_KEY
 NEXT_PUBLIC_SUPABASE_URL=https://PROJECT_ID.supabase.co
+NEXT_PUBLIC_API_URL=http://localhost:8080
```

Comment:
server url shouldn't be exposed with `NEXT_PUBLIC_` since FE side don't need context to this endpoint

Reply:
Thanks for catching this, fixed [here](https://github.com/is442oop/portfolio-analyzer-web-ui/pull/29/commits/2fca2d79e8c9271df9e8263a37ce7c297365f776)